pkgs/by-name/aw/aws-nuke/package.nix

```
@@ -42,12 +42,12 @@ buildGoModule rec {
       --zsh <($out/bin/aws-nuke completion zsh)
   '';
 
-  meta = with lib; {
-    description = "Nuke a whole AWS account and delete all its resources";
-    homepage = "https://github.com/rebuy-de/aws-nuke";
-    changelog = "https://github.com/rebuy-de/aws-nuke/releases/tag/v${version}";
-    license = licenses.mit;
-    maintainers = with maintainers; [ grahamc ];
+  meta = {
+    description = "Remove all the resources from an AWS account";
+    homepage = "https://github.com/ekristen/aws-nuke";
+    changelog = "https://github.com/ekristen/aws-nuke/releases/tag/v${version}";
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ grahamc ];
     mainProgram = "aws-nuke";
```

```suggestion
    mainProgram = "aws-nuke";
    # fork/exec exe/mockgen: exec format error
    # resources/autoscaling_mock_test.go:1: running "../mocks/generate_mocks.sh": exit status 1
    broken = !stdenv.buildPlatform.canExecute stdenv.hostPlatform;
```
Cross is broken due to use of mockgen.